docs/01-app/03-api-reference/05-config/01-next-config-js/turbopackPersistentCaching.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: turbopackPersistentCaching
+description: Learn how to enable Persistent Caching for Turbopack builds
+version: canary
+---
+
+## Usage
+
+Turbopack Persistent Caching enables turbopack to share state across multiple `next dev` or `next build` commands. When enabled, Turbopack will save and restore data to the `.next` folder between builds, which can greatly speed up builds.
```

Comment:
Asking for my own clarity here first

- A sequence of `next dev` -> `next dev` hits the Persistent Cache
- A sequence of `next dev` -> `next build`, wouldn't hit the cache for the build step, right? Or I guess, it'd miss it because it wouldn't hit production assets?
- A sequence `next build` -> `next dev` -> `next build` for the 3rd command, would hit cached elements from the 1st, or?

If so, maybe we can reword

```suggestion
Turbopack Persistent Caching lets multiple `next dev` runs share state, and likewise for `next build` runs. When enabled, Turbopack stores data in the `.next` folder and reuses it between matching command runs, greatly speeding them up.
```

Reply:
next dev and next build can share the cache.  The cache hit rates won't always be super high between these two but they aren't nothing. (e.g. most `resolve` results and many basic parsing tasks are identical)

Reply:
documented this.  might be overkill but i assume other people will have the question as well.